config-api/src/main/java/org/keycloak/config/TransactionOptions.java

```diff
@@ -5,17 +5,26 @@
 
 public class TransactionOptions {
 
-    public static final Option<Boolean> TRANSACTION_XA_ENABLED = new OptionBuilder<>("transaction-xa-enabled", Boolean.class)
+    public static final Option<Boolean> DB_XA_ENABLED = new OptionBuilder<>("db-xa-enabled", Boolean.class)
```

Comment:
Please, change the option. IMO, it belongs to the database namespace.

Another option is to keep the option name (`transaction-xa-enabled`). Perhaps this makes more sense?

Reply:
+1 to leaving the name as is, then it's also not a breaking change :) will change